share/icons/meson.build

```diff
@@ -1,4 +1,4 @@
-icons_dir = get_option('datadir') / 'icons/hicolor'
+icons_dir = datadir_path / 'icons/hicolor'
```

Comment:
Why did you replace `get_option('datadir')` with `datadir_path` everywhere? `datadir_path` seems to always be equal to `get_option('datadir')`.